app/views/topics/index.html.erb

```diff
@@ -47,8 +47,8 @@
 
 <div>
   <% if user_signed_in? %>
-    <span class="hidden-xs pull-right"><%= link_to t(:open_new_discussion, default: "Open Discussion"), new_topic_path, class: 'btn btn-primary' %></span>
+    <span class="hidden-xs pull-right"><%= link_to t(:start_discussion, default: "Start Discussion"), new_topic_path, class: 'btn btn-primary' %></span>
```

Comment:
And on a related note, I feel that scattering the `default: 'Start Discussion'` all over the place is not DRY, and could be avoided if we _required_ that key to be in place? The app config should take care of using the `EN` translation as default if not present in the current language, and - to avoid the risk of "no translation found" blank strings - we could use Rails' `config.action_view.raise_on_missing_translations = true` setting to crash and bring these to our attention during testing.